src/Tizen.Multimedia/AudioManager/AudioSampleFormat.cs

```diff
@@ -0,0 +1,41 @@
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Tizen.Multimedia
+{
+    public enum AudioSampleFormat
```

Comment:
put this enum into AudioIOEnums.cs or seperate them all like this.

Reply:
I think that no need to put this enum to audio io's.
There's a chance to add more enum values in audio manager regardless of audio io's, I think currently it is fine to divide each file as is.